### PR TITLE
Task management

### PR DESCRIPTION
**Pull request by lima-fox**

### CODE REVIEW

File: src/AppBundle/Controller/TaskController.php [hunk 1]
```diff
@@ -97,11 +97,19 @@ public function deleteTaskAction(Task $task)
 
         if ($task->getUser() == null)
         {
-            $em = $this->getDoctrine()->getManager();
-            $em->remove($task);
-            $em->flush();
+            if ($this->getUser()->getRoles() == ['ROLE_ADMIN'] || $this->getUser()->getRoles() == ['ROLE_SUPER_ADMIN'])
```
Comment (hi2ks) [inline src/AppBundle/Controller/TaskController.php]: This wont work if later you introduced a new common ROLE shared among users.
To check if a value is in an array, please consider using : in_array()

**Review by hi2ks — CHANGES_REQUESTED** [review]

File: src/AppBundle/Controller/TaskController.php (hunk 1 above)
Comment (lima-fox) [reply src/AppBundle/Controller/TaskController.php]: ok commit fix pr

File: src/AppBundle/Controller/TaskController.php [hunk 2]
```diff
@@ -97,7 +97,8 @@ public function deleteTaskAction(Task $task)
 
         if ($task->getUser() == null)
         {
-            if ($this->getUser()->getRoles() == ['ROLE_ADMIN'] || $this->getUser()->getRoles() == ['ROLE_SUPER_ADMIN'])
+            if ($this->getUser()->getRoles() == in_array('ROLE_ADMIN', $this->getUser()->getRoles()) ||
```
Comment (hi2ks) [inline src/AppBundle/Controller/TaskController.php]: NO

**Review by hi2ks — CHANGES_REQUESTED** [review]

File: src/AppBundle/Controller/TaskController.php (hunk 2 above)
Comment (lima-fox) [reply src/AppBundle/Controller/TaskController.php]: 😅

**Pull request merged**